platform/darwin/src/MGLNetworkConfiguration.m

```diff
@@ -14,6 +20,7 @@ + (instancetype)sharedManager {
     dispatch_once(&onceToken, ^{
         _sharedManager = [[self alloc] init];
         _sharedManager.sessionConfiguration = nil;
+        _sharedManager.events = [NSMutableDictionary dictionary];
```

Comment:
This should really be part of `MGLNetworkConfiguration`'s initializer - let's move into an `init`